### PR TITLE
To Do list: interactive list

### PR DESCRIPTION
**Pull request by ragangithub**
- Updated  HTML file
- Updated  CSS file
- Updated  JS file

### CODE REVIEW

File: src/modules/task-collection.js [hunk 1]
```diff
@@ -46,18 +62,35 @@ export default class TaskCollection {
...
 
-  displayTasks = () => {
+    this.tasks = this.tasks.filter((task) => task.completed !== true);
+    this.tasks.forEach((task, i) => {
+      task.index = i + 1;
+    });
+
+    localStorage.setItem('tasks', JSON.stringify(this.tasks));
+    this.displayTasks();
+  }
```
Comment (OyePriscilla) [inline src/modules/task-collection.js]: - [ ] Please according to the project requirement, kindly create a separate file and import it as a module. It should contain methods related to the **status updates**. ✔️

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @ragangithub 👏 👏, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Add event listener to the checkbox ✔️ 
Implement a function for the "Clear all completed" button ✔️ 
Updated array of items in local storage ✔️ 

## Required Changes ♻️

- [ ] @ragangithub  It would be great if you can make your PR description to be more descriptive. PR description should be a detailed summary of what your PR is all about. 👍 

## Optional suggestions

- [ ] For easy usage of your app, kindly add the start script to the `package.json` file. ℹ️ **Though I observed that you're starting your app with `npm run dev` command**

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ragangithub** [comment]
- created a new JS file  and imported it as a module .

**Review by prolajumokeoni — APPROVED** [review]
Hi @ragangithub 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
  - All changes requested have been implemented 🥇  


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**